test/integration/test_ibm_job.py

```diff
@@ -182,9 +182,7 @@ def test_retrieve_running_error_jobs(self):
     def test_retrieve_queued_cancelled_jobs(self):
         """Test client side filtering with queued and cancelled jobs."""
         statuses = ["QUEUED", "CANCELLED"]
-        job_list = self.provider.backend.jobs(
-            backend_name=self.sim_backend.name, limit=3, status=statuses
-        )
+        job_list = self.provider.backend.jobs(limit=3, status=statuses)
```

Comment:
This test sometimes takes a very long time to run if we also filter by backend (because it can take a while to find queued/cancelled jobs). We are only testing filtering by status here so we can remove the backend filter.